Stripe/STPPaymentContext.m

```diff
@@ -205,6 +201,13 @@ - (void)setDelegate:(id<STPPaymentContextDelegate>)delegate {
     }];
 }
 
+- (void)setPrefilledInformation:(STPUserInformation *)prefilledInformation {
+    _prefilledInformation = prefilledInformation;
+    if (prefilledInformation.shippingAddress) {
+        self.shippingAddress = prefilledInformation.shippingAddress;
```

Comment:
ah yeah, good point – I'll check if we already have a shipping address set, also set `shippingAddressNeedsVerification = YES;`.

I think I still prefer using a `shippingAddressNeedsVerification` flag here rather than using a nil `_shippingAddress`, since things get complicated with the `shippingAddressNeedsVerification` configuration option and `customer.shipping`, and I think it's better to be more explicit.